### PR TITLE
Update doc for windows timezones support

### PR DESCRIPTION
**Pull request by Stromweld**
https://github.com/chef/chef/blob/master/lib/chef/resource/timezone.rb the timezone resource also supports windows and this adds mention of that as well as a link to a list of windows timezone names to use.

### Description

Current resource description sounds like it only supports Linux and Mac when resource also supports windows. This also adds a link to windows timezone names.

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by tas50** [comment]
@Stromweld we'll want to get this same description update actually applied to the resource code in chef/chef itself. We've been generating these pages using that code.

File: chef_master/source/resource_timezone.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ timezone resource
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_timezone.rst>`__
 
-Use the **timezone** resource to change the system **timezone** on Linux and macOS hosts. Timezones are specified in tz database format, with a complete list of available TZ values here https://en.wikipedia.org/wiki/List_of_tz_database_time_zones.
+Use the **timezone** resource to change the system **timezone** on Windows, Linux, and macOS hosts. Timezones are specified in tz database format, with a complete list of available TZ values for linux and macOS here https://en.wikipedia.org/wiki/List_of_tz_database_time_zones and for windows here https://ss64.com/nt/timezones.html.
```
Comment (mjingle) [inline chef_master/source/resource_timezone.rst]:
```suggestion
Use the **timezone** resource to change the system **timezone** on Windows, Linux, and macOS hosts. Timezones are specified in tz database format, with a complete list of available TZ values for Linux and macOS here: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones and for Windows here: https://ss64.com/nt/timezones.html.
```

**Comment by Stromweld** [comment]
@tas50 do you want me to open a PR in chef/chef for that or are you going to do that?

**Comment by tas50** [comment]
Could you open a chef/chef PR with the final text

**Comment by Stromweld** [comment]
yes I can

**Comment by Stromweld** [comment]
opened PR https://github.com/chef/chef/pull/8018

**Comment by mjingle** [comment]
@Stromweld Thank you for your contribution! Because this is a PR from a remote fork, our set-up means that we had to include your work to chef-web-docs through other means. Please see https://github.com/chef/chef-web-docs/pull/1784

**Pull request closed without merging**